manifest-examples/image-multi-version.json

```diff
@@ -0,0 +1,82 @@
+{
+  "schemaVersion": "3",
+  "mediaType": "application/vnd.oci.distribution.manifest.v1+json",
+  "objects": [
+    {
+      "artifactType": "application/vnd.oci.image",
+      "version": "2",
+      "components": [
+        {
+          "type": "blob",
+          "mediaType": "application/vnd.oci.config",
+          "version": "2",
+          "descriptor": {
+            "mediaType": "application/vnd.oci.image.config.v1+json",
+            "size": 7023,
+            "digest": "sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7"
+          }
+        },
+        {
+          "type": "blob",
+          "mediaType": "application/vnd.oci.layer",
+          "version": "2",
+          "descriptor": {
+            "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+            "size": 32654,
+            "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0"
+          }
+        },
+        {
+          "type": "blob",
+          "mediaType": "application/vnd.oci.layer",
+          "version": "2",
+          "descriptor": {
+            "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+            "size": 16724,
+            "digest": "sha256:3c3a4604a545cdc127456d94e421cd355bca5b528f4a9c1905b15da2eb4a4c6b"
+          }
+        },
+        {
+          "type": "blob",
+          "mediaType": "application/vnd.oci.layer",
+          "version": "2",
+          "descriptor": {
+            "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+            "size": 73109,
+            "digest": "sha256:ec4b8955958665577945c89419d1af06b5f7636b4ac3da7f12184802ad867736"
+          }
+        },
+        {
+          "type": "blob",
+          "mediaType": "application/vnd.oci.layer.encrypted",
+          "version": "2",
+          "descriptor": {
+            "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",
```

Comment:
Is +encrypted going to get bikesheded as an example? Should we use a different example of different layers?